app/services/exports/export_request_service.rb

```diff
@@ -84,11 +87,18 @@ def compute_item_headers
         end
       end
 
+      # Include inactive items or items that are otherwise not in any item_requests
+      @organization_items.each do |item|
+        if item_names.exclude?(item.name)
+          item_names << item.name
```

Comment:
Can `item_names` be a set instead of an array? Then this is automatically handled.